charters/IngestSupport/charter.md

```diff
@@ -0,0 +1,100 @@
+# Ingest Support
+
+## Description
+
+The Ingest Support team acts as a liaison between labs who want to contribute data to the Human Cell Atlas (HCA) and the Data Coordination Platform (DCP). This team works with data contributing labs, to understand each lab's data and ensure the experiments are well-represented in the platform and to support the lab writing data to the platform, working with them to resolve any issues in the process.
+
+## Definitions
+
+*Data contributor* - any user who uploads data and metadata into the DCP
+
+*Data wrangler* - a member of the ingest support team who supports to data contributors wanting to upload data into the DCP
+
+*Submission* - the grouping of data and metadata that makes sense to a data contributor. For example, a data contributor might typically upload all the data at once for a publication they have in preparation. This grouping does not necessarily correspond to how data is stored or displayed in the DCP and how it should be displayed to a data consumer
+
+*Data* - raw data files from an experiment such as fastq files for sequencing experiments or tiff stacks for imaging experiments
+
+*Metadata* - descriptive information about what projects, samples and experiments are and how they were collected or conducted
+
+*New data types* - experimental data types which either can't already be ingested by the DCP or have only recently (~6m - 1yr) started to be ingested by the system
+
+*Established data types* - experimental data types which have a history of submission and downstream use in the DCP and there are already many datasets of this type represented in the platform
+
+## Objectives
+
+- Supporting data contributors uploading data and metadata to the DCP
+- Helping data contributors fix errors with their data and metadata before it can be submitted to the DCP
+- Ensuring a data contributor's experiment can be accurately represented in the DCP
+- Working with the data contributor to propose extensions to the HCA metadata schema if additional metadata fields are required
```

Comment:
Regarding _Working with_ - Does the contributor help with the proposal -or- is Ingest Support proposing extensions to address the needs of the contributor based on your deep understanding of their data?

Such as:

_Proposing enhancements to the HCA DCP metadata schema if additional metadata fields are required by the data contributor_

Reply:
This will be a variable process depending on the newness of the data, the level of understanding of the data contributor and how engaged the contributor is in the process.

The SpaceTX folks are actively engaged  and working with Zina and Matt to define and review the spec though I don't know if they have actually made direct metadata requests in the git sense

Many of our data contributors don't know what they could be giving us and in those cases, it is very much wrangler driven.

I don't want to write this in a way which precludes either scenario. Would two statements to reflect this situation be better.

I have made an update, I am not sure if it is better or not